tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -1,9 +1,10 @@
 import os
 from .common import get_user_client, validate_cluster, \
-    random_test_name, AWS_SSH_KEY_NAME
+    random_test_name, AWS_SSH_KEY_NAME, wait_for_cluster_delete
```

Comment:
can we add a line for each import from `.common`? this multi import line is getting a bit long.

Reply:
This is within the 80 character limit. Do you think that should be ok ?

Reply:
Yeah I noticed it is in conformance of the line length but the import line is just getting multiline split and not sorted alphabetically, and an all caps variable in the middle. It just looks a bit odd.